Scripts/ensure_license

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
+
+require 'find'
+
+# Read and normalize the license text by removing extra whitespace
+license_text = "/*\n" + File.read('LICENSE') + "*/\n"
+normalized_license = license_text.gsub(/\s+/, ' ').strip
+
+Find.find('.') do |path|
+  next unless File.file?(path) && path.end_with?('.swift') && !path.end_with?('Package.swift')
+
+  # Read the current content of the Swift file
+  content = File.read(path)
+
+  # Extract the existing license part from the file, if present
+  if content =~ /\A\s*\/\*.*?\*\//m
+    existing_license = $&.gsub(/\s+/, ' ').strip
+  else
+    existing_license = ""
+  end
+
+  # Check if the existing license matches the normalized expected license
+  if existing_license != normalized_license
+    puts "License missing or incorrect in file: #{path}"
+    exit 1
+  else
+    puts "✔️ #{path}"
+  end
+end
```

Comment:
This change allows us to remove the `git diff --name-only --exit-code` from the CI workflow and exit in the script itself.

It also allows for tabs/spaces in the LICENSE code comments since swiftformat will automatically indent it.